.github/workflows/linux-ci.yml

```diff
@@ -59,22 +59,10 @@ jobs:
           cache-key: python
         timeout-minutes: 10
 
-      - name: Install poetry
-        run: |
-          set -eux
-          export POETRY_HOME="$HOME/.poetry"
-          mkdir -p "$POETRY_HOME"
-          curl -sSL https://install.python-poetry.org | python3 - --version=${{ env.poetry-version }}
-          echo "$POETRY_HOME/bin" >> "$GITHUB_PATH"
-
-      - uses: actions/setup-python@c4e89fac7e8767b327bbad6cb4d859eda999cf08  # pin v4.1.0
-        id: setup-python
+      - uses: ./.github/actions/setup-poetry
```

Comment:
Renaming the action `setup-python-poetry` would be more explicit (and it's already the name in action.yml)

Reply:
done